System.IO.Ports/SerialPort.cs

```diff
@@ -406,15 +406,14 @@ public int ReadTimeout
         /// <exception cref="IOException">The port is in an invalid state. -or- An attempt to set the state of the underlying
         /// port failed. For example, the parameters passed from this <see cref="SerialPort"/>
         /// object were invalid.</exception>
-        /// <exception cref="ArgumentOutOfRangeException">The <see cref="WriteTimeout"/> value is less than zero and not equal
-        /// to <see cref="Timeout"/>.</exception>
+        /// <exception cref="ArgumentOutOfRangeException">The read time-out value is less than zero and not equal to <see cref="Timeout.Infinite"/>.</exception>
```

Comment:
```suggestion
        /// <exception cref="ArgumentOutOfRangeException"> The read time-out value is less than zero, or is greater than <see cref="Int32.MaxValue"/>.</exception>
```

Reply:
One of our goals is to be in sync with the full framework and API and only deviate when absolutely required. The above it's [coming from .NET](https://learn.microsoft.com/en-us/dotnet/api/system.io.ports.serialport.readtimeout?view=dotnet-plat-ext-6.0&viewFallbackFrom=net-6.0), on purpose. And I don't see a good reason to introduce this breaking change...